packages/hydrooj/src/handler/training.ts

```diff
@@ -206,20 +206,20 @@ class TrainingEditHandler extends Handler {
     @param('dag', Types.Content)
     @param('pin', Types.UnsignedInt)
     @param('description', Types.Content)
+    @param('maintainer', Types.NumericArray, true)
     async post(
-        domainId: string, tid: ObjectId,
-        title: string, content: string,
-        _dag: string, pin = 0, description: string,
+        domainId: string, tid: ObjectId, title: string, content: string,
+        _dag: string, pin = 0, description: string, maintainer: number[] = [],
```

Comment:
Optional parameters should always be at the end, maybe `description: string, pin = 0, maintainer: number[] = []` or `description: string = ''`?